### ISSUE TITLE
Inline config: disable indent rules

### ISSUE THREAD

**Issue by acalvo** [linthtml/linthtml#246]
Hi again 😅

I have the following HTML code:

```html
<p>
  <span>Lorem ipsum
    dolor sit amet</span>
</p>
```

With this LintHTML config:

```json
{
  "indent-style": "spaces",
  "indent-width": 2
}
```

And it throws this error:

```
✔ Found 1 files
✔ Files analyzed

File: index.html
Config file: /Users/alberto/Desktop/linthtml/.linthtmlrc.js
   3:19   error  Incorrect indentation for `span` beginning at    indent-style 
                 L3:C19. Expected `<\span>` to be at an                        
                 indentation of 2 but was found at 3.                          

✖ 1 problem (1 error, 0 warning)
```

That's OK. I understand the error and agree with it. But I really need it to be like that in that specific case. So I tried disabling the rule (with inline config). I tried several things like:

```html
<p>
  <!-- linthtml-configure indent-style="false" -->
  <span>Lorem ipsum
    dolor sit amet</span>
</p>
```

But no luck at all. The error keeps showing. Am I doing something wrong or is it a bug?

Thanks!

**Comment by KamiKillertO** [comment]
Ok, I've reproduced. 🤔 For some reason, it's no working.

**Comment by KamiKillertO** [comment]
The `indent-*` rules ignore the inline config unless it's on top of the file.
This works:
```html
<!-- linthtml-configure indent-width="false" -->
<!-- linthtml-configure indent-style="false" -->
<p>
  <span>Lorem ipsum
    dolor sit amet</span>
</p>
```
Contrary to the other rules, the `indent-*` rules are not run for each HTML node but only with the first one (`p` in this case).
I need to refactor these rules to make them work like the others. And until then, I'll have a warning message in an inline configuration doc to states this limitation.

**Comment by acalvo** [comment]
Thanks for the workaround! Putting it on top of the file does the trick 😊
